eradiate/radprops/rad_profile.py

```diff
@@ -1083,11 +1087,24 @@ def eval_sigma_a_ckd(self, *bindexes: Bindex) -> pint.Quantity:
         ds = eradiate.data.open(
             category="ckd_absorption", id="afgl1986-us_standard-10nm_test"
         )
+        # evaluate H2O and O3 concentrations
+        h2o_concentration = compute_column_mass_density(
+            ds=self.thermoprops, species="H2O"
+        )
+        o3_concentration = compute_column_number_density(
+            ds=self.thermoprops, species="O3"
+        )
+
         z = to_quantity(self.thermoprops.z_layer).m_as(ds.z.units)
         return ureg.Quantity(
             [
                 ds.k.sel(bd=(bindex.bin.id, bindex.index))
-                .interp(z=z, kwargs=dict(fill_value=0.0))
+                .interp(
+                    z=z,
+                    H2O=h2o_concentration.m_as(ds.H2O.units),
+                    O3=o3_concentration.m_as(ds.O3.units),
```

Comment:
I'd rather favour the dict syntax to access dataset members [since it seems to be the most basic way to do so](http://xarray.pydata.org/en/stable/user-guide/data-structures.html#dataset-contents).

```suggestion
                    H2O=h2o_concentration.m_as(ds["H2O"].units),
                    O3=o3_concentration.m_as(ds["O3"].units),
```

Reply:
I've gotten used to access data set members using the properties instead of the dict syntax because if I want to refactor things later, and also because the code is more compact, but I guess I could give the dict syntax another try 😃 In this particular case, the dict syntax is probably a better choice because the coordinate names match the names of the molecules that are in the thermo-physical properties data set.